### DISCUSSION TITLE
Integrations page: different colors for same badge

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16870] [Dashboard]
Have a look at this part:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/f372df4e-8405-4139-b6b6-ce7b04f006c5)

Here 3 integrations of different types:
-- custom
-- depending on a cloud
-- custom + depending on a cloud

Each tile case an icon (badge) describing a type.
And "cloud" part of a badge has different colors.
Proposals:
-- either use 2 separate badges for "custom" & "cloud" of different colors;
-- or use same color.

**Comment by spacegaier** [comment]
I disagree with your proposal. Using two badges will use more space without providing any real benefit. Having the orange color for custom integrations is also sensible. Therefore, the fact that custom and cloud are then in one orange badge makes sense to me as well.

**Comment by ildar170975** [comment]
Isn't it strange that SAME "cloud" icon has TWO different colors?

**Comment by ildar170975** [comment]
Fixed in 2023.7 - exactly as I like it!

![image](https://github.com/home-assistant/frontend/assets/71872483/85286963-01d0-49c3-a759-b12b858b57a9)
